.cargo/config

```diff
@@ -1,13 +1,59 @@
 [alias]
-# TODO: --all-targets fails for benches
-t = "test --all-features --tests --lib --bins --examples"
-b = "build --all-features --all-targets"
-lint = "clippy --all-features --all-targets -- -D warnings"
-nostd = "check --lib --bins --no-default-features --target=thumbv7m-none-eabi"
-perf = "bench --all-features --bench benchmark --package"
+t = """
+    hack test
+    --all-targets
```

Comment:
Can we remove these, so that these aliases only trigger one build process?

Reply:
Removing this doesn't prevent the duplication of messages, so let's leave it in for now.